src/containers/Header/Header.test.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Header from './Header';
```

Comment:
I know why the tests are failing. We need to do named exports of our containers i.e. `export class Header extends Component`. Then in the test file, the Header needs to be destructured as it's imported i.e. `import { Header } from './Header'`. This needs to be done because the default export of the containers are of a `connect` invocation. The test files just need the class and not the Redux connected class.

Reply:
That was the solution, resolved!